examples/AffinePlaneCurve.jl

```diff
@@ -0,0 +1,124 @@
+module AffinePlaneCurveModule
+
+using Oscar, Markdown
+
+export AffinePlaneCurve, AffinePlaneCurve_degree, AffinePlaneCurve_Jacobian, AffinePlaneCurve_tangent, AffinePlaneCurve_issmoothpoint, AffinePlaneCurve_union
+
+####################################################
+#
+#We follow the lecture notes by Andreas Gathmann: https://www.mathematik.uni-kl.de/~gathmann/en/curves.php
+#
+####################################################
+mutable struct AffinePlaneCurve{S}
```

Comment:
What is the parameter S for? It doesn't seem to be used at all in the type, and I didn't spot it being used below (though I haven't read very far yet).

Reply:
Edit: oh I see, it is for the field type. You can just use T here as well if you like and also write "T <: FieldElem" as far as I recall.